### PR TITLE
[PURIFY] Remove certs and PKCS12 files temporarily

### PR DESCRIPTION
**Pull request by kavilla**
### Description
Removed the generated keys from Elastic. Since we did not own those
and do not have the private key for the cert authority then it will
be safer.

This should not have any impact on runtime, might impact security
related to x-pack with demo certs but none of the certs where
for production. Otherwise, this is strictly for testing.

To be clear, this can be emptied out because it is ONLY for demo
and testing purposes. The demo security can be accomplished by
using:

https://github.com/opensearch-project/security/blob/main/tools/install_demo_configuration.sh

Eventually we should take the certs from that file and copy those over but will
just ignore the tests for now.

Signed-off-by: Kawika Avilla <kavilla414@gmail.com>
 
### Issues Resolved
Purposefully didn't created one.

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass
- [x] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: packages/osd-opensearch/src/integration_tests/cluster.test.js [hunk 1]
```diff
@@ -265,7 +265,7 @@ describe('#start(installPath)', () => {
     await expect(cluster.start()).rejects.toThrowError('OpenSearch has already been started');
   });
 
-  it('sets up SSL when enabled', async () => {
+  it.skip('sets up SSL when enabled', async () => {
```
Comment (ananzh) [inline packages/osd-opensearch/src/integration_tests/cluster.test.js:268]: Why it is skipped? Can we add a comment / todo here?

**Review by ananzh — COMMENTED** [review]
nice. just one comment

**Review by ananzh — APPROVED** [review]
nice. thanks rocky for this.

**Review by mihirsoni — APPROVED** [review]
LGTM !!

**Pull request merged**